deploy/ansible/initial-setup.yml

```diff
@@ -14,12 +14,18 @@
     - role: goshimmer-node
```

Comment:
What is this file used for? Manual setup of the machines outside of the automated deployments?

Reply:
Yes. I run this playbook from my local machine to set up the whole stack. 
I decided to have it in the repo so the whole environment would be available as a code